conda/reporters.py

```diff
@@ -6,27 +6,62 @@
 
 from __future__ import annotations
 
+import logging
+import sys
+
+from .base.constants import (
+    DEFAULT_CONSOLE_REPORTER_BACKEND,
+    DEFAULT_JSON_REPORTER_BACKEND,
+)
 from .base.context import context
+from .exceptions import CondaError
+
+logger = logging.getLogger(__name__)
 
 
 def render(data, style: str | None = None, **kwargs) -> None:
-    for settings in context.reporters:
-        reporter = context.plugin_manager.get_reporter_backend(settings.get("backend"))
-        output = context.plugin_manager.get_reporter_output(settings.get("output"))
+    """
+    Used to render output in conda
+
+    The output will either be rendered as "json" or normal "console" output to stdout.
+    This function allows us to configure different reporter backends for these two types
+    of output.
+    """
+    if context.json:
+        backend = context.reporter_backends.json.get(
+            "backend", DEFAULT_JSON_REPORTER_BACKEND
+        )
+    else:
+        backend = context.reporter_backends.console.get(
+            "backend", DEFAULT_CONSOLE_REPORTER_BACKEND
+        )
```

Comment:
To be clear, `console` will remain the fallback/default reporter backend, basically?